traefik/templates/_podtemplate.tpl

```diff
@@ -76,6 +76,15 @@
           hostIP: {{ $config.hostIP }}
           {{- end }}
           protocol: {{ default "TCP" $config.protocol | quote }}
+        {{- if $config.http3 }}
+        {{- if and $config.http3.enabled $config.hostPort }}
+        {{- $http3Port := default $config.exposedPort $config.http3.advertisedPort }}
```

Comment:
`$http3Port` does not seem to be used here?

Reply:
Oops, you're right. Too much copy paste from me.

```suggestion
```